cogment_verse/web/web_app/src/controls/AtariPongPzFeedback.jsx

```diff
@@ -0,0 +1,185 @@
+// Copyright 2021 AI Redefined Inc. <dev+cogment@ai-r.com>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { useCallback, useEffect, useState } from "react";
+import { cogment_verse } from "../data_pb";
+import { useDocumentKeypressListener } from "../hooks/usePressedKeys";
+import { useRealTimeUpdateV2 } from "../hooks/useRealTimeUpdate_v2";
+import { Button } from "../components/Button";
+import { FpsCounter } from "../components/FpsCounter";
+import { KeyboardControlList } from "../components/KeyboardControlList";
+import { Switch } from "../components/Switch"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faThumbsUp, faThumbsDown, faMeh } from "@fortawesome/free-solid-svg-icons";
+import { serializePlayerAction, DType, Space } from "../utils/spaceSerialization";
+
+const ACTION_SPACE = new Space({
+    box: {
+        low: {
+            // Only the dtype and shape of the lower bound are used to serialize the action
+            dtype: DType.DTYPE_FLOAT32,
+            shape: [1],
+        },
+    },
+});
+
+export const AtariPongPzHfbEnvironments = ["environments.pettingzoo_atari_adapter.HumanFeedbackEnvironment/pettingzoo.atari.pong_v3"];
+export const AtariPongPzFeedback = ({ sendAction, fps = 30, actorClass, observation, ...props }) => {
+    const [paused, setPaused] = useState(false);
+    const [humanMode, setHumanMode] = useState(true);
+    const [selectedFeedback, setFeedback] = useState(2);
+    const [humanTurn, setHumanTurn] = useState(false);
+    const [playerPos, setPlayerPos] = useState('');
+    const [playerNameDisplay, setPlayerNameDisplay] = useState(false);
+    const playerName = observation?.gamePlayerName;
```

Comment:
should be computed from the environment specs.

Reply:
I wanted the name of current player is playing at the current time step or tick id